src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/ParseContainerPropertiesTests.cs

```diff
@@ -100,4 +100,21 @@ public void CanOnlySupplyOneOfTagAndTags()
         Assert.True(logs.Errors.Count > 0);
         Assert.Equal(logs.Errors[0].Code, ErrorCodes.CONTAINER2008);
     }
+
+    [Fact]
```

Comment:
As it, this should be 
```suggestion
    [DockerDaemonAvailableFact]
```

Reply:
That's a great point - I was looking at the ComputeContainerConfig target (and the ParseContainerProperties task) and none of them require Docker at all so I figured this Fact would be enough. I can try checking what other tests don't require it in this file.